libs/service-portal/regulations-admin/README.md

```diff
@@ -16,6 +16,20 @@ Get fresh AWS credentials, and then open six (6) terminal windows.
 
 Once everything is running, open <http://localhost:4200/minarsidur/reglugerdir-admin> and enjoy.
 
+## Downloading PDFs
+
+Too be able to download PDF files, also start:
+
+`yarn start download-service`
+
+PDFs are downloaded by:
+
+* Requesting a download via `downloadRegulation` with the regulation id as input
+* If the requested regulation is a draft regulation, returns a object with a `download-service` url and a boolean that marks it as a download-service file
+* `download-service` fetches all required data from the admin db and posts to the API server which returns a base64 of the PDF
+
+For non-draft regulations, it should go straight to the public API for the PDF, this is not implemented.
```

Comment:
```suggestion
```

We won't **ever** need this for non-draft regulations.

Those are hosted, and served directly from https://files.reglugerd.is/pdf/*